baseline.py

```diff
@@ -25,7 +25,7 @@
 
 def main(baseline_file_path, files, unwanted_pixels=[]):
 
-    data_stream = event_stream(file_list=files)
+    data_stream = event_stream(files)
```

Comment:
I think here we should just keep the name `file_list` not `files`

Reply:
I just realize both a wrong.

event_stream happily eats any sequence of *paths* (not files), this includes
 * lists,
 * tuples
 * any iterable, which spits out paths, like e.g. `glob.iglob("*.fits.fz")`

So while I think `files` beats `file_list` here ... `paths` would be best.

----

We use `file_list` many of the other scripts ... but I think `paths` would be best.

Reply:
Ok, but then we should edit it also in:
`files=args['<files>'],` 
and
`def main(baseline_file_path, files, unwanted_pixels=[]): `

Reply:
yep.

just in the help, we should stick to "<files>" I think, since it seems to be common in linux man pages to talk about "FILES" instead of "PATHS", c.f. [man page of rm](https://linux.die.net/man/1/rm).

So using "FILE" and "PATH" as as synonyms is fine on the command-line. But in source code there is a difference between a file and its path.

Reply:
Ah another option would be `urls`

Reply:
I think what is most important is the consistency of variable names